dot-net-sdk/EppoClient.cs

```diff
@@ -233,6 +234,51 @@ public static EppoClient Init(EppoClientConfig eppoClientConfig)
     }
 
 
+    /// <summary>Selects a Bandit Action from the list of <paramref name="actions"/>, if applicable, for the given <paramref name="flagKey"/> and subject context.
+    /// <param name="flagKey">The flag or bandit key to lookup.</param>
+    /// <param name="subject">The subject's identifier and a collection of attributes.</param>
+    /// <param name="actions">A list of actions without contextual attributes.</param>
+    /// <param name="defaultValue">Default flag variation.</param>
+    /// <example>
+    /// For Example:
+    /// <code>
+    /// var client = EppoClient.GetInstance();
+    /// var subject = new ContextAttributes("subjectKey")
+    /// {
+    ///     ["age"] = 30,
+    ///     ["country"] = "uk",
+    ///     ["pricingTier"] = "1"  // NOTE: Deliberately setting to string causes this to be treated as a categorical attribute
+    /// };
+    ///
+    /// var actions = new string[] { "adidas", "nike", "Reebok" };
+    ///
+    /// var result = client.GetBanditAction(
+    ///     "flagKey",
+    ///     subject,
+    ///     actions,
+    ///     "defaultValue"
+    /// );
+    /// </code></example>
+    /// </summary>
+    public BanditResult GetBanditAction(string flagKey,
+                                        ContextAttributes subject,
+                                        string[] actions,
+                                        string defaultValue)
+    {
+        try
+        {
+            var actionContexts = actions.ToDictionary(action=>action, action=> new ContextAttributes(action));
+            return GetBanditDetail(flagKey, subject, actionContexts, defaultValue);
```

Comment:
👍

Reply:
This will throw an exception if duplicate keys. I think that's ok as we'd just return the default in the catch block, but if easy consider deduplicating the actions first.

Reply:
good catch. super easy fix.